flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -89,21 +89,35 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                                     + "seconds suffix, daily expression's formation is startTime-endTime, such as 9:30:30-10:50:20, when exclude from 9:30:30-10:50:20 in Monday and Thursday "
                                     + "we can express it as 9:30:30-10:50:20 && * * * ? * 2,5");
 
-    public static final ConfigOption<Double> TARGET_UTILIZATION =
-            autoScalerConfig("target.utilization")
+    public static final ConfigOption<Double> UTILIZATION_TARGET =
+            autoScalerConfig("utilization.target")
                     .doubleType()
                     .defaultValue(0.7)
                     .withFallbackKeys(oldOperatorConfigKey("target.utilization"))
                     .withDescription("Target vertex utilization");
 
-    public static final ConfigOption<Double> TARGET_UTILIZATION_BOUNDARY =
-            autoScalerConfig("target.utilization.boundary")
+    public static final ConfigOption<Double> UTILIZATION_TARGET_BOUNDARY =
+            autoScalerConfig("utilization.target.boundary")
                     .doubleType()
                     .defaultValue(0.3)
                     .withFallbackKeys(oldOperatorConfigKey("target.utilization.boundary"))
                     .withDescription(
                             "Target vertex utilization boundary. Scaling won't be performed if the processing capacity is within [target_rate / (target_utilization - boundary), (target_rate / (target_utilization + boundary)]");
 
+    public static final ConfigOption<Double> TARGET_UTILIZATION_MAX =
+            autoScalerConfig("utilization.target.max")
```

Comment:
```suggestion
            autoScalerConfig("utilization.max")
```

`job.autoscaler.utilization.max` is the option key we discussed in FLINK-36836.